js/src/relay/WalletLinkRelay.ts

```diff
@@ -517,8 +573,8 @@ export class WalletLinkRelay implements WalletLinkRelayAbstract {
     message: RelayMessage,
     callWebhook: boolean
   ): Observable<string> {
-    const secret = this.session.secret
-    return new Observable<string>((subscriber) => {
+    let secret = this.session.secret
+    return new Observable<string>(function (subscriber) {
```

Comment:
Can you change this back? Doesn't look like secret is being reassigned.

Reply:
Blockhan wellet address: 90aa4a2b-3fc3-449c-9492-a4142b3fbda6